design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Azure Simple
+draft: true
+date: 2019-05-18T13:09:18.252Z
+author:
+  name: Rachel Weil
+  github-username: hxlnt
+sourceUrl: https://github.com/microsoft/cobalt/
+tags:
+  - app-insights
+  - app-gateway
+  - azure
```

Comment:
Should -azure and -monitoring be on separate lines? Looks like they should be collapsed to say "azure-monitoring".

Reply:
Left these separate to be as broad as possible. This way, they will filter under "Azure," "Monitoring," and "Azure Monitoring."